### PR TITLE
Clarify smooth=true criteria

### PR DESCRIPTION
**Pull request by chcunningham**
See #155

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/media-capabilities/pull/156.html" title="Last updated on Jun 29, 2020, 3:30 PM UTC (3183cd9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/media-capabilities/156/a9bdf79...3183cd9.html" title="Last updated on Jun 29, 2020, 3:30 PM UTC (3183cd9)">Diff</a>

### CODE REVIEW

**Comment by chcunningham** [comment]
@cconcolato @chrisn

I made a small edit and updated the encoding smooth=true definition to mostly match. While it is reasonable that encoding is limited by the rate at which the encoder is fed, this is also true of decoding and seemed extraneous.

File: index.bs [hunk 1]
```diff
@@ -471,7 +471,7 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
             </li>
             <li>
               <dfn for='HdrMetadataType' enum-value>smpteSt2094-40</dfn>,
-              representing the dynamic metadata type defined by
```
Comment (vi-dot-cpp) [inline index.bs:474]: Thanks for fixing this. May I ask whether there is a tool like `git cl format` for W3C spec edits?

**Comment by chrisn** [comment]
LGTM too

File: index.bs (hunk 1 above)
Comment (chcunningham) [reply index.bs:474]: Not that I'm aware of. I just caught this by accident when cleaning up my own trailing spaces.

**Pull request merged**